components/Nav/RightNav.jsx

```diff
@@ -0,0 +1,89 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+/*This code is for the style of right navbar which is changed when screen size is mobile. Also the if statement inside style components is for underline transition for the nav options when the screen is not mobile size*/
+const Ul = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-flow: row nowrap;
+  font-size: 0.5em;
+  li {    
+    padding-right: 3em;
+    padding-top: 1em;
+    cursor: pointer;    
+  }    
+  ${ function ({open}){
+    if (open==false) {
+      return (
+        `.hover-underline-animation {
+          display: block;
+          position: relative;    
+        }
+        
+        .hover-underline-animation:after {
+          content: '';
+          position: absolute;
+          width: 60%;
+          transform: scaleX(0);
+          height: 2px;
+          bottom: 0;
+          left: 0;
+          background-color: #fff;
+          transform-origin: bottom right;
+          transition: transform 1s ease-out;
+        } 
+        
+        .hover-underline-animation:hover:after {
+          transform: scaleX(1);
+          transform-origin: bottom left;
+        }               
+        `
+        
+      );
+    }else{
+      return ``;
```

Comment:
- Add the underline animation in the burger menu links as well

Reply:
but the inspiration site(https://experiencewelcome.com/) didnt have any animation for burger menu. Will do if you want to.